Python/pystate.c

```diff
@@ -2,6 +2,7 @@
 /* Thread and interpreter state structures and their interfaces */
 
 #include "Python.h"
+#include "pycore_emscripten_trampoline.h"  // _Py_EmscriptenTrampoline_Init()
```

Comment:
Nit: the include list is alphabetically sorted ;)

Reply:
For some reason I was thinking that e came before c. I almost just responded to say that it *is* sorted.